src/main/java/com/noticemedan/map/viewmodel/CanvasView.java

```diff
@@ -64,55 +82,89 @@ public void paint(Graphics _g) {
 			}
         }
 
-		System.out.println("Range size: " + forest.rangeSearch(viewArea).size());
+		Stopwatch stopwatchRangeSearch = new Stopwatch();
+		List<OsmElement> result = forest.rangeSearch(viewArea, zoomLevel);
```

Comment:
Nogen specifik grund til at tage det ud til en variabel?

Reply:
Yeah, det er redundant og bør fjernes.

Reply:
Jeg bliver nødt til at tage den ud for at teste dens performance med et stopwatch.